UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -102,6 +125,9 @@ private void Update()
     
     private void OnControllerColliderHit(ControllerColliderHit hit)
     {
+        //updates where the collider is hiting;
+        collisionPoint = hit.point;
+        collisionPoint = (collisionPoint - transform.position);
```

Comment:
Just asking so that I can learn, but is this figuring out the collisionPoint relative to the character? (As opposed to the global coordinates of the collision.)

Reply:
Not OP but yeah, it's figuring out where it did collide, in this case, relative to the GO's position